src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -538,10 +551,9 @@ else if (conf.repair_session_space.toMebibytes() > (int) (Runtime.getRuntime().m
         long valueInBytes = conf.native_transport_max_frame_size.toBytes();
         if (valueInBytes < 0 || valueInBytes > Integer.MAX_VALUE)
         {
-            throw new ConfigurationException(String.format("%s must be positive value < %dB, but was %dB",
+            throw new ConfigurationException(String.format("%s must be positive value <= %dB, but was %dB",
                                                            "native_transport_max_frame_size",
```

Comment:
nit: Do we actually need a substitution for `native_transport_max_frame_size`, or can we just inline it?